### ISSUE TITLE
Decorative image is exposed to assistive technologies - ITS alerts information icon

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9687]
# Motivation

Siteimprove is reporting "Decorative image is exposed to assistive technologies". This is a WCAG 2.1 A compliance issue related to the info icon wherever alerts are rendered.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r67

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/994335/A11y/Page?pageId=25126162339&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r67/failed/V0dWWFdyZlg=;ZGI1OGU1YTI=
- https://my2.siteimprove.com/Inspector/994335/A11y/Page?pageId=25127527051&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r67/failed/L2pNRkNzaSs=;YmFlMGM5ODk=
- https://my2.siteimprove.com/Inspector/994335/A11y/Page?pageId=71276337401&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r67/failed/VlNIbGZsdUo=;MjIwMWMwOWY=

Example
- https://its.uiowa.edu/alerts
- https://its.uiowa.edu/alerts/status-history
- https://its.uiowa.edu/services/campus-data-network-connectivity

**Decorative image is exposed to assistive technologies**
Issue
1.1.1: Non-text Content
This image or SVG has been marked as decorative, but it remains exposed to assistive technologies.

This could be because:

the image has a text alternative or other ARIA attribute (ARIA attributes contain "instructions" for assistive technologies), or
the image is focusable (meaning it can still be reached by keyboard)
